entity-service/build.gradle.kts

```diff
@@ -15,12 +15,12 @@ tasks.integrationTest {
 dependencies {
   implementation(project(":entity-service-impl"))
 
-  implementation("org.hypertrace.core.grpcutils:grpc-server-utils:0.4.0")
-  implementation("org.hypertrace.core.grpcutils:grpc-client-utils:0.4.0")
+  implementation("org.hypertrace.core.grpcutils:grpc-server-utils:0.6.1")
+  implementation("org.hypertrace.core.grpcutils:grpc-client-utils:0.6.1")
   implementation("org.hypertrace.core.serviceframework:platform-service-framework:0.1.28")
   implementation("org.hypertrace.core.documentstore:document-store:0.6.0")
 
-  runtimeOnly("io.grpc:grpc-netty:1.36.1")
+  runtimeOnly("io.grpc:grpc-netty:1.40.1")
   constraints {
     runtimeOnly("io.netty:netty-codec-http2:4.1.61.Final") {
```

Comment:
I was hoping that upgrading gprc would help 🤣

Reply:
as you saw, grpc doesn't bring in a new enough version (otherwise this would have succeeded in the previous rev). The reason being that constraints aren't overrides, they're minimums, same as any other dependency declaration.